xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/drawer.less

```diff
@@ -20,87 +20,133 @@
 @xwiki-drawer-menu-category-bg:          @xwiki-drawer-menu-item-seperator-color;
 @xwiki-drawer-menu-category-color:       lighten(@xwiki-drawer-menu-item-color, 10%);
 
-.drawer-overlay {
-  z-index: 5200;
-}
-
 .drawer--right .drawer-nav {
   right: -@xwiki-drawer-width;
 }
 
-.drawer-nav {
+
+
+#tmDrawer.drawer-nav {
   background-color: @xwiki-drawer-bg;
   width: @xwiki-drawer-width;
-  z-index: 5300;
+  z-index: 5200;
+  position: fixed;
+  top: 0;
+  overflow: hidden;
+  height: 100vh;
+  right: -@xwiki-drawer-width;
+  transition: right .6s cubic-bezier(0.190, 1.000, 0.220, 1.000), visibility .6s;
+  &.closed {
+    // We use visibility here to avoid breaking the animations.
+    visibility: hidden;
+  }
+  &.opened {
+    right: 0;
+  }
 
-  // Brand =================================================================
-  .drawer-brand {
-    background-color: @xwiki-drawer-brand-bg;
-    border-bottom: 1px solid @xwiki-drawer-brand-border;
-    font-weight: 400;
-    line-height: inherit;
-    padding: 20px 16px;
+  & + .drawer-overlay {
+    position: fixed;
+    display: none;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    opacity: 0.2;
+    background-color: #fff - @xwiki-page-content-bg;
+  }
 
-    a {
-      display: block;
-      color: @xwiki-drawer-brand-link-color;
-      padding: 0;
+  &.opened + .drawer-overlay {
+    display: block;
+  }
 
-      &:hover, &:focus {
-        color: @xwiki-drawer-brand-link-hover-color;
-        text-decoration: none;
-      }
+  & .drawer-menu {
+    position: relative;
+    z-index: 5300;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+
+    & .drawer-menu-item {
+      font-size: 1rem;
+      display: block;
+      padding: .75rem;
+      text-decoration: none;
+      color: #222;
     }
 
-    // Avatar styling
-    img {
-      border-radius: 50%;
-      float: left;
-      height: 50px;
-      width: 50px;
+    & .drawer-menu-item:hover {
+      color: #555;
```

Comment:
Same here, why is this color hardcoded?

Reply:
:+1: Fixed in https://github.com/xwiki/xwiki-platform/pull/2282/commits/3bea63ccc02cc7327faf52f4513ecb50dfcd1321